config/default-disco.js

```diff
@@ -58,5 +58,5 @@ module.exports = {
   // stand-in for the ui-tour.
   useUiTour: false,
 
-  po2jsonFuzzyOutput: true,
+  po2jsonFuzzyOutput: false,
```

Comment:
I'm not too familiar with this setting. Is it used only by the extractor script? Does it cause an error if we are missing a translation? If so, will it only cause an error in production or in dev too? Since this is in the default config, I'm wondering if we need something that allows missing strings to fall back to English on dev.